src/config.rs

```diff
@@ -169,6 +169,8 @@ pub struct EnableMiddleware {
 #[derive(Debug, Clone, Deserialize, Serialize)]
 pub struct Mailer {
     pub smtp: Option<SmtpMailer>,
+    #[serde(default)]
+    pub stub: bool,
```

Comment:
notice -- this is the only thing out of the testing chain that has no "testing" cfg attribute,

Reply:
done